### ISSUE TITLE
TypeError: unsupported operand type(s) for /: 'dict' and 'float'

### ISSUE THREAD

**Issue by rudametw** [carderne/signal-export#142]
Hi,

I'm trying to export all of my Signal conversations but I've run into the following error, it's probably something simple :

```python
/home/wrudamet/.local/share/pipx/venvs/signal-export/lib/python3.12/site-packages/sigexport/utils.py:16 in dt_from_ts

  13
  14
  15 def dt_from_ts(ts: float) -> datetime: 
❱ 16 return datetime.fromtimestamp(ts / 1000.0) 
  17
  18
  19 def parse_datetime(input_str: str) -> datetime:

 ╭───────────────────────── locals ─────────────────────────╮
   ts = {'high': 350, 'low': -1436384941, 'unsigned': True}
 ╰──────────────────────────────────────────────────────────╯
 TypeError: unsupported operand type(s) for /: 'dict' and 'float'
 ```
 
This is just the tail. The trace is much larger and includes conversation info and the calling functions. Let me know what to do or what else you might need to help with this bug.

Best regards

**Comment by carderne** [comment]
Should be fixed by this commit:
https://github.com/carderne/signal-export/commit/4dd484a1a2f679b1a09112b27d0b5f5bb2a2beeb

**Comment by carderne** [comment]
And released in v3.2.0

**Comment by rudametw** [comment]
Thanks. I confirm, it works great!
